testplan/web_ui/testing/package.json

```diff
@@ -16,13 +16,15 @@
     "enzyme-adapter-react-16": "1.6.0",
     "eslint": "5.10.0",
     "eslint-plugin-react": "7.11.1",
+    "npm": "^6.10.1",
     "react": "16.6.0",
     "react-copy-html-to-clipboard": "6.0.4",
     "react-custom-scrollbars": "4.2.1",
     "react-dom": "16.6.0",
     "react-portal-tooltip": "2.4.0",
     "react-scripts": "1.1.1",
     "react-test-renderer": "16.6.0",
+    "react-vis": "^1.11.7",
```

Comment:
TODO: we need to onboard react-vis internally before we can merge this PR. I have opened a ticket.